app/templates/index.html

```diff
@@ -45,19 +45,24 @@
 			      		<input type="checkbox" id="pdf" name="pdf" checked="checked">PDF
 			      	</label>
 			  	</div>
+				<div id="error" class="option-error"> Please select atleast one option! </div>
 			  	<div class="form-group">
 			  		<label for="inputFile">Upload a CSV file to generate badges</label>
 			  		<input type="file" id="inputFile" name="file">
 			  	</div>
+				<div id="error" class="csvfile-error"> Please upload a CSV file! </div>
 			  	<div class="text-center"><strong>OR</strong></div>
 			  	<div class="form-group">
-			  		<label for="textArea">Enter data manually</label>
+				<label for="textArea">Enter data manually</label>
 					<textarea class="form-control" id="textArea" name="csv"></textarea>
-			  	</div>
-			  	<div class="form-group">
-						<div class="text-center guide">
-                 Check out our <a href="/guide" target="_blank">User Input Guide</a>
-						</div>
+				</div>
+				<div id="error" class="no-data-error"> Please upload a csv file or enter data! </div>
+				<div id="error" class="csv-error"> Please fill data in proper format
+					<div class="text-center guide">
```

Comment:
@djmgit  How about changing `Please fill data in proper format` it to `Do follow the data guidlines. For more info:` ?

Reply:
Else Nice Work @djmgit!!